inst/scripts/run_CoNGA.py

```diff
@@ -0,0 +1,116 @@
+from pathlib import Path
+import scanpy as sc
+import conga
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+from conga import util
+import scipy
+import os
+import sys
+from conga.preprocess import retrieve_tcrs_from_adata
+from conga.preprocess import read_adata
+from conga.preprocess import normalize_and_log_the_raw_matrix
+from conga.tcrdist.tcr_distances import TcrDistCalculator
+tcrdist_calculator = TcrDistCalculator('rhesus')
```

Comment:
shouldnt this be read from a variable, so we have the ability to process other species?

Reply:
maybe we dont actually need this?

Reply:
Thanks, I addressed in my latest commit.